Xamarin.Flex/Flex.cs

```diff
@@ -557,7 +555,7 @@ static void layout_item(Item item, float width, float height)
 					float basis = child.Basis.Length;
 					if (child.Basis.IsRelative)
 						basis *= (layout.vertical ? height : width);
-					child.Frame[layout.frame_size_i] = basis;
+					child.Frame[layout.frame_size_i] = basis - child.MarginThickness(layout.vertical);
```

Comment:
this removes the margin when using `basis` from the main-axis dimension

Reply:
How will this map to the code ported from flex lib? shouldn't at least we add a note that we mad this change?

Reply:
the code never mapped